cypress/integration/dib-travel/other/helpers/add-generic-product.ts

```diff
@@ -0,0 +1,26 @@
+import { GenericProduct } from '@cy/models';
+
+export const addGenericProduct = (genericProduct: GenericProduct) => {
+  cy.get('home dib-generic-product-form input[name=serviceName]').type(genericProduct.serviceName);
+
+  cy.get('home dib-generic-product-form input[name=bookingReferenceId]').type(genericProduct.bookingReferenceId);
+  cy.get('home dib-generic-product-form .checkmark').click();
+  cy.get('home dib-generic-product-form h5').contains('Start date').click();
+  cy.get('.cdk-overlay-container ui-date-picker-calendar-v2 span').contains(1).click();
```

Comment:
What "1" represents here? Same "2" below? It's not clear from the code. Maybe you can put this in `const` with a meaningful name, what do you think?

Reply:
It represents dates in date-picker, but for sure, I agree with you to put this in `const`.